daemons/execd/remoted_pidone.c

```diff
@@ -94,102 +94,116 @@ find_env_var_name(char *line, char **first, char **last)
 }
 
 static void
-load_env_vars(const char *filename)
+load_env_vars(void)
 {
+    static const char *const filename = "/etc/pacemaker/pcmk-init.env";
```

Comment:
If the filename is always going to be the same, could we just make it a #define and not have to use this ridiculous type?

Reply:
Oh yeah, this is a combination of (a) remnant of another approach and (b) late night delirium